### ISSUE TITLE
modeButtonColor black

### ISSUE THREAD

**Issue by jminutaglio** [DDraiman1990/DroidTimeSelection#4]
Changing the modeButtonColor doesn't seem to work, the button is there, but black on black...

```
let droidSelection = DroidTimeSelection()
        
droidSelection.config.timeFormat = .twelve
droidSelection.config.okButtonColor = .green
droidSelection.config.modeButtonColor = .purple
```

Results in a green OK button, but not a purple modeButton.

**Comment by DDraiman1990** [comment]
Thanks for bringing this up. I'll investigate further.

**Comment by jminutaglio** [comment]
ok, add'l info:

it's not that the tint isn't applying its that the images aren't there - I'm using a pod install.  
<img width="231" alt="Screen Shot 2021-03-17 at 11 17 22 PM" src="https://user-images.githubusercontent.com/49879291/111568335-f3234e00-8776-11eb-82db-50b7c0c73d2f.png">

I'd suggest:
- change min ver to iOS 13
- use the SF Symbols for clock & keyboard

ex: in DroidTimeSelction.swift; line 60:
 ```
   
    // MARK: - Private Properties
    private var currentConfig: DroidTimeSelectionConfiguration = .init()
    private let keyboardIcon = UIImage(systemName: "keyboard")
    private let clockIcon = UIImage(systemName: "clock")
    private var mode: HourSelectionMode = .clock
    private var currentTime: Time = .init()
    
```

Other approaches work obviously, but SF Symbols kind of makes everything easier imo.

TY again for making this project!  I appreciate you.

**Comment by DDraiman1990** [comment]
Hey. Library is updated with the new refactor. Please check and confirm this bug went away for you.

**Comment by jminutaglio** [comment]
Confirmed: this is resolved in the 2.0.0 refactor. TY!
